### PR TITLE
94/t/STORY-3309/Migrate to Adwords api v201509

### PR DESCRIPTION
**Pull request by ibiziiac-sv**
WIP

### CODE REVIEW

**Comment by ibiziiac-sv** [comment]
@sherriman please review

File: Gemfile [hunk 1]
```diff
@@ -2,5 +2,5 @@ source "https://rubygems.org"
 
 gemspec
 
-gem 'savon_zuora', :git => 'git@github.com:raduoprita/savon', :branch => 'savon_0_9_9'
-gem 'nori_savon', :git => 'git@github.com:raduoprita/nori.git', :branch => 'nori_1_1_5'
+gem 'savon_zuora', :git => 'git@github.com:Invoca/savon', :branch => '91/t/STORY-3309/update_httpi_and_wasabi_dependencies'
```
Comment (codyjca) [inline Gemfile]: Small nit, the branch should be 94/t/ instead of 91/t.

**Comment by codyjca** [comment]
Just one comment, @sherriman if you have time to take a look.

File: Gemfile (hunk 1 above)
Comment (ibiziiac-sv) [inline Gemfile]: This branch was created back in sprint 91, but there is a PR opened to the `compatibility_mode` branch. So this line will point to `compatibility_mode` again after the PR in `savon_zuora` will be merged.

Comment (codyjca) [inline Gemfile]: Ack.  I thought it was a brand new branch, thanks!

Comment (sherriman) [inline Gemfile]: See comments about using SHA.

**Comment by sherriman** [comment]
@ibiziiac-sv 
- You should bump the version, right?
- See comments about identifying gems by SHA.

**Comment by ibiziiac-sv** [comment]
@sherriman now gems are referenced by SHA.
You mean to bump the version of `active_zuora` gem? I think it would have a massive impact on Invoca's integration with zuora.

**Comment by sherriman** [comment]
@ibiziiac-sv Looks good.  The only open question is the version.

Regarding the version, I meant that we should change the version we list in our gem.
- Edit the gem file to bump the lowest version.
- So (if) it says 2.0.3 now it should say 2.0.4

Then in Web App I think a bundle update `active_zuora` would show the latest version.  I guess you would have to move the SHA too since it will have moved with that commit.

**Comment by sherriman** [comment]
module ActiveZuora
  VERSION = "2.0.4"
end

**Comment by ibiziiac-sv** [comment]
thanks @sherriman

**Pull request merged**